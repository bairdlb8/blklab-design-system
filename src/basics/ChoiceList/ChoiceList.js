import React from 'react'
import PropTypes from 'prop-types'

import './ChoiceList.scss'

const ChoiceList = ({className=''}) => {
  return <div>ChoiceList</div>
}

ChoiceList.propTypes = {}

export default ChoiceList
