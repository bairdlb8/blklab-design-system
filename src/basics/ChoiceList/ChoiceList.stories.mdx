import { Meta, Story, Props, Preview } from '@storybook/addon-docs/blocks'

import ChoiceList from './ChoiceList'

<Meta title="Basics / ChoiceList" component={ ChoiceList } />

# ChoiceList

## Add some text here

export const Template = (args) => {
  return <ChoiceList {...args} />
}

<Preview withToolbar>
  <Story name="Primary" args={ {} }>
    {Template.bind({})}
  </Story>
</Preview>

<Props of={ ChoiceList } />
