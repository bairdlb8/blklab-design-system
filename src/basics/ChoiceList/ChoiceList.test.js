import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { axe, toHaveNoViolations } from 'jest-axe'
import ChoiceList from './ChoiceList'

expect.extend(toHaveNoViolations)

describe('<ChoiceList />', () => {
  describe('accessibility', () => {
    it('should not fail any accessibility tests', async () => {
      const { container } = render(<ChoiceList />)
      expect(await axe(container)).toHaveNoViolations()
    })
  })

  it('should take a snapshot', () => {
    const { asFragment } = render(<ChoiceList />)

    expect(asFragment(<ChoiceList />)).toMatchSnapshot()
  })
})
