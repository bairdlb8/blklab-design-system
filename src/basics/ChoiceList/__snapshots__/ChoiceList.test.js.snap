// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChoiceList /> should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    ChoiceList
  </div>
</DocumentFragment>
`;
